test/cache.c

```diff
@@ -9853,15 +9853,6 @@ check_flush_cache__flush_op_eviction_test(H5F_t *file_ptr)
  * Programmer:    John Mainzer
  *              1/12/05
  *
- * Modifications:
```

Comment:
Why were these lines removed?

Reply:
Modification lines in general are going away. It was easier to delete them than correct them.

Reply:
I know you've been doing that and I heard your reasoning.  It's just sometime I found some good explanation in these Modifications notes very helpful.  I'm not pushing to keep them though...